src/main/java/org/folio/consortia/domain/entity/SharingRoleEntity.java

```diff
@@ -0,0 +1,41 @@
+package org.folio.consortia.domain.entity;
+
+import java.util.Objects;
+import java.util.UUID;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.folio.consortia.domain.entity.base.AuditableEntity;
+
+@Getter
```

Comment:
As with the previous Keycloak PRs, many of these Lombok annotations can be replaced with `@Data` an `@RequiredArgsConstructor`